tests/test_for_suite/conftest.py

```diff
@@ -40,6 +40,14 @@ def model_file(small_file, tmpdir_factory):
 
 
 @pytest.fixture(scope="session")
+def invalid_file(invalid_model, tmpdir_factory):
```

Comment:
The roles of the `invalid_model` and `invalid_file` fixtures are a bit unclear to me. The first seem to return the path to the invalid model, while the latter returns the path to a tmp copy of it. Is the tmp copy needed because tests could make modifications to the file? I would maybe consider naming the fixtures a bit differently or adding docstrings to make their functions clear.

Reply:
I frankly just replicated the implementation I already found, so as far as I understand this also applies to `small_file` and `model_file`. Perhaps, @Midnighter could shed some light as to why this is the way it is?

Reply:
> > To combat both, would I have to use `git submodules` then? And what about the drawbacks/ implications that has?
> > https://stackoverflow.com/questions/4659549/maintain-git-repo-inside-another-git-repo
> 
> No, I wouldn't add it as a submodule. I think it's simplest to store the repo as a zip/tarball and in the fixture just uncompress it to a temporary folder.

OK that seems reasonable.